packages/websocket/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@atproto/websocket",
```

Comment:
nit: would call this `websocket-server`

Reply:
yeah it's actually a client lib, but i'm down with `@atproto/ws-client`